repl/src/test/scala/org/apache/spark/repl/ReplSuite.scala

```diff
@@ -84,6 +85,7 @@ class ReplSuite extends SparkFunSuite {
       settings = new scala.tools.nsc.Settings
       settings.usejavacp.value = true
       org.apache.spark.repl.Main.interp = this
+      in = SimpleReader()
```

Comment:
This was giving an NPE in 2.12. I think this little hack fixes the issue for purposes of this test.